iina/Extensions.swift

```diff
@@ -173,6 +183,12 @@ extension NSMenu {
   }
 }
 
+extension String {
```

Comment:
Could you explain what this does exactly? From what it looks like you're trying to `filter` the `String`–am I correct?